tests/test_build.py

```diff
@@ -49,3 +53,23 @@ def test_install_dist():
         specifier.contains(installed_versions[package])
         for package, specifier in meta_specifiers.items()
     )
+
+
+@pytest.mark.latest_runtime
+def test_latest_coiled():
```

Comment:
This is a little gnarly, but works. To minimize local test suite failures, I'm marking this to only run on our latest `coiled-runtime` build. I think this is okay for now, we can always relax that contsraint later if needed